pandas/tests/extension/test_floating.py

```diff
@@ -24,12 +24,22 @@
 from pandas.tests.extension import base
 
 
-def make_data():
+def make_data(with_nas: bool = True):
+    if with_nas:
```

Comment:
umm can't you just parameterize over `[pd.NA, np.nan]`?
otherwise these blocks look exactly the same

Reply:
alternatley just
`na_value = pd.NA if with_nas else np.nan`

Reply:
updated